packages/babel-parser/src/parser/expression.js

```diff
@@ -556,6 +556,7 @@ export default class ExpressionParser extends LValParser {
   ): N.Expression {
     const state = {
       optionalChainMember: false,
+      eventualMember: false,
```

Comment:
Is this still used now that you have removed the `eventual` property from the AST?

Reply:
No, it isn't.  Thanks!